### DISCUSSION TITLE
Clickable/Toggleable Timestamp

### DISCUSSION THREAD

**Discussion by barraIhsan** [tomasklaen/uosc#884] [Q&A] [answered]
In the vanilla mpv, you can click the timestamp thing and it can do 2 things
1. Toggle decimals if we click the first (left) timestamp
![image](https://github.com/tomasklaen/uosc/assets/57800056/b3ae0142-e85a-45dd-ae33-5aad8d7ee33a)
![image](https://github.com/tomasklaen/uosc/assets/57800056/91f112e8-6505-42ff-aba2-cd62cc59b4e8)

2. Toggle Total and Remaining
![image](https://github.com/tomasklaen/uosc/assets/57800056/17e9e50e-b5e0-41cf-9cdb-730a258fc363)
![image](https://github.com/tomasklaen/uosc/assets/57800056/c209b43f-579b-427e-8232-41bc1a1b447a)

Can we do that in uosc? I would love to enable the decimals

**Comment by xfzv** [comment, marked as answer]
> Can we do that in uosc? I would love to enable the decimals

https://github.com/tomasklaen/uosc/blob/e6a5fd981dadcc2a0d30ded64b7bcbd4481692cd/src/uosc.conf#L179-L180

**Reply by barraIhsan** [reply]
oh, sorry, didn't caught that before 😅
